osu.Game/Screens/Play/SquareGraph.cs

```diff
@@ -201,43 +209,49 @@ public ColumnState State
 
             public Column()
             {
-                Width = WIDTH;
+                Width = cube_size;
             }
 
             protected override void LoadComplete()
             {
-                for (int r = 0; r < cubeCount; r++)
+                // Create the boxes in reversed order to start iterations at the bottom
+                for (int r = (int) cubeCount - 1; r >= 0; r--)

```

Comment:
Also I don't like that this creation is in reversed order - was trying to decode how fillActive worked and was confused because it iterated from 0 which I thought was the start of the bar.